Cargo.toml

```diff
@@ -13,9 +13,19 @@ Simple numerics. This crate contains basic arbitrary-sized integer,
 rational, and complex types.
 """
 
-[dependencies]
-rustc-serialize = "0.3.13"
-rand = "0.3.8"
+[dependencies.rustc-serialize]
+version = "0.3.13"
+optional = true
+
+[dependencies.rand]
+version = "0.3.8"
+optional = true
+
+[features]
+
+rational = []
```

Comment:
I'm not sure. Maybe rational doesn't need a feature, I didn't look how big it was. I expected complex to be small. We can feature gate them all?